kaybee/internal/tasks/merge.go

```diff
@@ -15,112 +15,88 @@ import (
 type MergeTask struct {
 	BaseTask
 	policy  model.Policy
-	sources []conf.Source
+	sources []conf.SourceV1
 }
 
 // NewMergeTask constructs a new MergeTask
 func NewMergeTask() (mergeTask *MergeTask) {
-
 	mt := MergeTask{}
 	return &mt
 }
 
 // WithPolicy sets the policy to be used to merge sources
 func (t *MergeTask) WithPolicy(p conf.Policy) *MergeTask {
+	log.Trace().Str("policy", string(p)).Msg("Adding")
 	switch p {
 	case conf.Strict:
-		if t.verbose {
```

Comment:
Note: the output produced with the verbose option is not logging, it is rather to be considered an integral part of the UI, it's the message for the user to be seen on the console. It can also go in the logfile if appropriate, but i consider the log a different view on the behaviour of the application